build.sbt

```diff
@@ -271,7 +271,7 @@ lazy val bench = project
   .settings(commonJvmSettings)
   .settings(
     libraryDependencies ++= Seq(
-      "org.scalaz" %% "scalaz-core" % "7.3.6"
+      "org.scalaz" %% "scalaz-core" % "7.3.7"
```

Comment:
Does one else feel that we can give up on this dep? Only one suite with few benchmarks depends on it.

Reply:
Benchmarks are all about comparing to useful reference points that are well understood. At the time, I think scalaz was just such a reference point, but I would have a hard time arguing that it remains so today. I would be in favor of removing these measurements.